docs/developer/serving/services/ingress-class.md

```diff
@@ -1,28 +1,21 @@
 # Configuring Services custom ingress class
 
-By default, when a Knative Service is created an Ingress Class (`ingress.class`) is automatically chosen for it based on the content of the `networking-config` ConfigMap located inside the `knative-serving` namespace. 
-
-The `networking-config` ConfigMap is configured when Knative Serving is installed, and if no other ingress class is specified, this defaults to `"istio.ingress.networking.knative.dev"`. 
-
-After it is configured in the `networking-config` ConfigMap, the `ingress.class` is used for all Knative Services that do not specify an `ingress.class` annotation.
-
-!!! warning
-    The ingress class value should only be modified during Knative Serving installation to avoid issues in your deployment.
```

Comment:
I removed this warning because changing the configmap just changes the default and doesn't affect existing installations.

Reply:
I'd maybe add a note to call that out then though, in case a user expected it to change existing installations?

Reply:
good point